featuretools/tests/primitive_tests/test_transform_features.py

```diff
@@ -1808,7 +1808,7 @@ def test_multiply_numeric_boolean():
 
     multiply_numeric_boolean = MultiplyNumericBoolean()
     for input in test_cases:
-        vals = pd.Series(input["val"])
+        vals = pd.Series(input["val"]).astype("Int64")
```

Comment:
If we don't covert here the input with a single `pd.NA` value ends up with a dtype of `object` which would never happen in real usage. An `object` input to the primitive also causes an error.